spec/ttt/core_spec.clj

```diff
@@ -1,14 +1,59 @@
 (ns ttt.core-spec
-  (:require [speclj.core :refer :all]
-            [ttt.core :refer :all]))
+  (:require [speclj.core :refer [describe context it should= should-invoke stub with-stubs]]
+            [ttt.core :as game]
+            [ttt.game-rules :as rules]
+            [ttt.ui :as ui]
+            [ttt.board :as board]
+            [ttt.player :as player]))
 
+(def current-state {:board [0 1 2 3 4 5 6 7 8] 
```

Comment:
should this be initial state since current state implies it changes as the game is being played?